crates/matrix-sdk-ui/src/timeline/inner/state.rs

```diff
@@ -381,7 +187,7 @@ impl TimelineInnerState {
                 if reaction_group.0.remove(&id).is_none() {
                     warn!(
                         "Tried to remove reaction by transaction ID, but didn't \
-                         find matching reaction in the related event's reactions"
+                     find matching reaction in the related event's reactions"
```

Comment:
nope, previous indentation was correct :-P

Reply:
Oh, did I accidentally undo the indent-fix commit? Thanks for catching.